manifests/filebeat.yaml

```diff
@@ -174,6 +174,15 @@ data:
       hosts: ["${LOGSTASH_URL}"]
       ssl: '{{ .ssl | default ( coll.Dict "enabled" "false" ) | data.ToJSON }}'
 {{ end }}
+{{ if eq .name "infra" }}
+{{ if ".kubernetes.apiServerExtraArgs.audit-log-path" }}
+    filebeat.inputs:
+      - type: log
+        enabled: true
+        paths:
+          - '{{ ".kubernetes.apiServerExtraArgs.audit-log-path" }}'
```

Comment:
There is an issue with this - the path variable replacement doesn't work and produces:

```yaml
filebeat.inputs:
  - type: log
    enabled: true
    paths:
      - '.kubernetes.apiServerExtraArgs.audit-log-path'
```

This is because the quoting required to make the variable `.kubernetes.apiServerExtraArgs.audit-log-path` valid yaml seems to be interpreted as a string by our templating system and not as a quoted variable.

However I am not sure how to fix it. All permutations of quoting result in either invalid yaml or some variation of the result above